hyperjumptech/monika
# Bug/134 fix slow log prints

## Pull Request

### PR Type
[x] fix for issue #134 

### Cause
1. Was using cli table formatting to prettify output
2. Was pretty but useless. yup pretty but useless.
3. Particularly slow for large logs

### Fix
1. removed cli-table formatting 
2. output history.db logs through pino
3. addded plainLog type for pino, for regular logging

### How to test
1. run monika for awhile on some probes
2. then request to see all the logs with the -l flag like: `bin/run -l`
3. output should immediately print
